lib/quantified/coder.rb

```diff
@@ -0,0 +1,22 @@
+module Quantified
+  class Coder
+    attr_reader :attribute, :unit
+
+    def initialize(attribute, unit)
+      @attribute = attribute
+      @unit = unit
+    end
+
+    def load(amount)
+      attribute.new(amount, unit) unless amount.nil?
+    end
+
+    def dump(quantity)
+      if quantity.is_a?(attribute)
```

Comment:
Why are you using `is_a?` here over `responds_to?`. I think this is good, but I want to make sure I understand.